### PR TITLE
Add "papa" board.

### PR DESCRIPTION
**Pull request by osenft**
```
Tested with TANGO_BOARD_FOR_TEST=papa

----------------------
`make prtest` summary:
----------------------
git rev-parse HEAD
93a47e57bf1e6122644d516ba8e1b981af91f2c9
git status
On branch add-papa-board
nothing to commit, working tree clean
```

### CODE REVIEW

File: papa/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+lto = true
+opt-level = "z"
+debug = true
+
+[dependencies]
+capsules = { path = "../third_party/tock/capsules" }
+kernel = { path = "../third_party/tock/kernel" }
+cortexm3 = { path = "../third_party/tock/arch/cortex-m3" }
+h1b = { path = "../h1b" }
+
```
Comment (jrvanwhy) [inline papa/Cargo.toml:39]: Please remove the blank lines at the end of the file.

File: papa/Xargo.toml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Google LLC
```
Comment (jrvanwhy) [inline papa/Xargo.toml:1]: I don't think we use Xargo anymore, so Xargo.toml can be removed.

File: papa/src/main.rs [hunk 3]
```diff
@@ -0,0 +1,465 @@
...
+use kernel::hil;
+use kernel::hil::entropy::Entropy32;
+use kernel::hil::rng::Rng;
+use kernel::mpu::MPU;
+
+use h1b::crypto::dcrypto::Dcrypto;
+use h1b::hil::flash::Flash;
+use h1b::nvcounter::{FlashCounter,NvCounter};
+use h1b::timels::Timels;
+use h1b::usb::{Descriptor, StringDescriptor};
```
Comment (jrvanwhy) [inline papa/src/main.rs:58]: Do you want USB to be part of Papa? It doesn't seem relevant.

File: papa/src/nvcounter_syscall.rs [hunk 4]
```diff
@@ -0,0 +1,191 @@
+// Copyright 2020 Google LLC
```
Comment (jrvanwhy) [inline papa/src/nvcounter_syscall.rs:1]: `NvCounter` may not be relevant either.

**Review by jrvanwhy — COMMENTED** [review]
I think several of these things should be in a crate shared between golf2 and papa. Specifically, AES, dcrypto, and personality seem like they are chip-specific syscall interfaces rather than board-specific interfaces.

File: papa/src/main.rs (hunk 3 above)
Comment (osenft) [reply papa/src/main.rs:58]: Oops, good catch. I meant to remove that.

**Pull request closed without merging**